Refresh.GameServer/Authentication/Token.cs

```diff
@@ -8,13 +8,26 @@ namespace Refresh.GameServer.Authentication;
 
 #nullable disable
 
+[JsonObject(MemberSerialization.OptIn)]
 public class Token : RealmObject
 {
     [PrimaryKey]
     public ObjectId TokenId { get; set; } = ObjectId.GenerateNewId();
     
     // this shouldn't ever be serialized, but just in case let's ignore it
-    [JsonIgnore] [XmlIgnore] public string TokenData { get; set; }
+    [XmlIgnore] public string TokenData { get; set; }
     
+    // Realm can't store enums, use recommended workaround
+    // ReSharper disable once InconsistentNaming (can't fix due to conflict with TokenType)
+    internal string _TokenType { get; set; }
```

Comment:
this should *probably* be an integer type, not a string, parsing an enum is slow